lib/postgres/postgres-pr/message.rb

```diff
@@ -18,6 +18,40 @@ def read_exactly_n_bytes(n)
   end
 end
 
+# Monkeypatch for OpenSSL::SSL::SSLSocket to support read_exactly_n_bytes
```

Comment:
I tested the login module with the bare minimum after switching to Rex::Socket and this didn't seem necessary any more. Should be able to remove it.